SecondYearFirstSemestr/Lazy/Lazy/LazyOneThread.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Lazy
+{
+    public class LazyOneThread<T>: ILazy<T>
+    {
+        private Func<T> supplier;
```

Comment:
```suggestion
        private Func<T?>? supplier;
```
нужно позволить supplier'у быть null, чтобы в конце его смог собрать сборщик мусора. А также позволить ему возвращать null